src/app/_directives/numbers-only.directive.ts

```diff
@@ -0,0 +1,20 @@
+import { Directive, ElementRef, HostListener, Input } from '@angular/core';
+
+@Directive({
+  selector: '[appNumbersOnly]'
+})
+export class NumberDirective {
+
+    @Input() numbericType = 'number';
```

Comment:
Can we remove line 8 as the variable numbericType is not used in the file

Reply:
numbericType removed from directive